### PR TITLE
fix: GP-255 GP-256 GP-238 form validation, age component

### PR DESCRIPTION
**Pull request by KoalaMango**

### CODE REVIEW

File: sass/themes/frost/components/_form.scss [hunk 1]
```diff
@@ -189,6 +200,35 @@ form {
...
+
+    &.active { 
+      background-color: $colour-blue;
+      color: $colour-white;
+      border: 0px;
+      padding: 23px 40px 19px;
+    }
+  }
+
+  .btn + .btn {
```
Comment (lebaz20) [inline sass/themes/frost/components/_form.scss]: i don't think this is a good practice

File: sass/themes/frost/components/_modal.scss [hunk 2]
```diff
@@ -1,35 +1,41 @@
+// ==========================================================================
+// Modal
+// ==========================================================================
 .modal__overlay {
```
Comment (lebaz20) [inline sass/themes/frost/components/_modal.scss]: Modal should align with form, has no slider and with no blue dots in the middle;

u16 modal

File: sass/themes/frost/components/_form.scss (hunk 1 above)
Comment (lebaz20) [inline sass/themes/frost/components/_form.scss]: @KoalaMango After you explained it to me, it's not a bad practice at all,
but may be adding a comment above will make it more clear

**Comment by KoalaMango** [comment]
fix pushed

**Pull request merged**